packages/react/src/pathfinder/pathfinder.tsx

```diff
@@ -22,19 +23,36 @@ import type { PathfinderProps } from "./pathfinder.types";
 
 export const Pathfinder = ({
   mode = "FULL",
-  schemaProps,
-  themeProps,
+  fetcherOptions,
+  schemaPollingOptions,
+  themeOptions,
 }: PathfinderProps) => {
   const schema = useSchemaStore.use.schema();
 
   useEffect(() => {
     // set the theme and handle overrides if provided
-    initializeTheme({ overrides: themeProps?.theme?.overrides });
+    initializeTheme({ overrides: themeOptions?.theme?.overrides });
+
+    //set our schema polling options if provided
+    if (schemaPollingOptions) {
+      const currentVals = schemaStore.getState().polling;
+      schemaStore.setState({
+        polling: {
+          ...(schemaPollingOptions.enabled
```

Comment:
Might be clearer if the condition was on the value rather than the object but not blocking